### PR TITLE
Fix Style for Home Page

### PR DESCRIPTION
**Pull request by Ibtisam-Hemmo**
Here is how it looks in Desktop screens
screencapture-localhost-3000-2023-07-19-16_13_02 (1)

### CODE REVIEW

File: src/components/filterBasedCard/FilterBasedCard.jsx [hunk 1]
```diff
@@ -1,43 +1,126 @@
...
+              fontWeight={{ xs: "800", md: "700" }}
+              lineHeight={{ xs: "16px", md: "52px" }}
+              fontSize={["14px", "20px", "35px", "40px"]}
+              color="#A53F64"
+            >
+              {cardInfo}
+            </Typography>
+            <IconBtn
+              iconsNames={iconName}
+              iconColor={"#A53F64"}
```
Comment (Rahaf-96) [inline src/components/filterBasedCard/FilterBasedCard.jsx:70]: use it from constants please

File: src/components/filterBasedCollection/FilterBasedCollection.jsx [hunk 2]
```diff
@@ -1,50 +1,79 @@
...
     {
-      cardId:1,
-      cardInfo:"limited edition products",
-      imageUrl:{
-        desktop:'https://images2.imgbox.com/4c/50/SdVeqLHx_o.png',
-        mobile:'https://images2.imgbox.com/17/94/7sNiYkmp_o.png'
-      }
+      cardId: 1,
+      cardTitle: "Lifestyle",
+      cardInfo: "Makeup Accessories from Top Brands",
```
Comment (Rahaf-96) [inline src/components/filterBasedCollection/FilterBasedCollection.jsx:12]: why did you change the card titles?

File: src/components/filterBasedCard/FilterBasedCard.jsx [hunk 3]
```diff
@@ -1,43 +1,126 @@
...
+          >
+            <Typography
+              textAlign="center"
+              height={{ xs: "32px", md: "104px" }}
+              width="100%"
+              margin={{ xs: "0 0 19px 0", md: "0 0 21px 0" }}
+              fontWeight={{ xs: "800", md: "700" }}
+              lineHeight={{ xs: "16px", md: "52px" }}
+              fontSize={["14px", "20px", "35px", "40px"]}
+              color="#A53F64"
```
[the remark this answers is not in the document]
Comment (Rahaf-96) [reply src/components/filterBasedCard/FilterBasedCard.jsx:64]: yes even so, as I see here the color is used twice in the same file.
it's a good practice either add it to the theme or have file for constants and add the colors there.
always think if your project gets larger, you need to make it easy to change.

File: src/components/filterBasedCard/FilterBasedCard.jsx [hunk 4]
```diff
@@ -1,43 +1,126 @@
...
-              <IconBtn iconsNames={iconName} iconColor={theme.palette.primary.main} iconJustifyContent={'flex-end'} iconSize={{xs:'20px' , md:'30px'}} iconBackgroundColor={'#17494D29'} pseudoType={'&:hover'} pseudoIconId={'EastIcon'} pseudoBackgroundColor={'#17494D29'} />
+            <IconBtn
+              iconsNames={iconName}
+              iconColor={theme.palette.primary.main}
+              iconJustifyContent={"flex-end"}
+              iconSize={{ xs: "20px", md: "30px" }}
+              iconBackgroundColor={"#17494D29"}
+              pseudoType={"&:hover"}
+              pseudoIconId={"EastIcon"}
+              pseudoBackgroundColor={"#17494D29"}
```
[the remark this answers is not in the document]
Comment (Ibtisam-Hemmo) [reply src/components/filterBasedCard/FilterBasedCard.jsx:116]: Yes,  it is a hex color value as the last two digits represents the opacity. xD

**Review by Rahaf-96 — APPROVED** [review]
